docs/using-flatpak.rst

```diff
@@ -0,0 +1,103 @@
+Using Flatpak
+=============
+
+This page provides an introduction to the most common commands needed to use Flatpak. It is not intended to be exhaustive or to cover all the options for each command (a full list of all the commands and their options can be found in `the command reference <command-reference.html>`_).
+
+.. note::
+  Flatpak commands can be run either per-user or system-wide. All the examples in this guide use the default system-wide behavior.
+
+Remotes
+-------
+
+Remotes are the repositories from which applications and runtimes can be installed.
+
+List remotes
+````````````
+
+To list the remotes that you have configured on your system, run::
+
+  $ flatpak remotes
+
+This gives a list of the existing remotes that have been added. The list indicates whether each remote has been added per-user or system-wide.
+
+Add a remote
+````````````
+
+Adding a remote allows you to search and list its contents, and to install applications from it. The most convenient way to add a remote is by using a ``.flatpakrepo`` file, which includes both the details of the remote and its GPG key::
+
+ $ flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
+
+Here, ``flathub`` is the local name that is given to the remote. The URL points to the remote's ``.flatpakrepo`` file. ``--if-not-exists`` prevents an existing remote from being overwritten.
+
+Remove a remote
+```````````````
+
+To remove a remote, run::
+
+ $ flatpak remote-delete flathub
+
+In this case, ``flathub`` is the remote's local name.
+
+Installing applications
+-----------------------
+
+Search
```

Comment:
It is worth adding a note here that for `search` to work that you must have at least once updated from a repo. `flatpak update --appstream` can download just the needed information.

Reply:
That's unfortunate and it would be better not to have to document that. :/

Reply:
I added a note about this in the latest commit.

Reply:
Agreed, ideally it would behave like `dnf`. If there is no info or its outdated try updating.

Reply:
> Agreed, ideally it would behave like dnf. If there is no info or its outdated try updating.

I've created an issue for this - https://github.com/flatpak/flatpak/issues/1339